mne/decoding/time_gen.py

```diff
@@ -109,6 +108,9 @@ class GeneralizationAcrossTime(object):
                 these predictions into a single estimate per sample.
 
         Default: 'cross-validation'
+    predict_shape : int
+        Dimensionality of clf.predict().
+        Default : 1
```

Comment:
mmh, no. The dimensionality output doesn't depend on fit() but on predict(), and predict can be multidimensional (if probabilistic output, decision_function and adhoc classifiers). We can keep the param private for now, or estimate it at the predict stage.